tests/42tags.pl

```diff
@@ -378,3 +382,49 @@ sub check_private_user_data {
          Future->done( 1 );
       });
    };
+
+test "Deleted tags appear in an incremental v2 /sync",
```

Comment:
If you add a `bug` argument here, then sytest will be OK about the fact the test fails.

    bug => "SYN-513",

You can then run tests presuming this bug to be fixed, by passing

    ./run-tests.pl ... --fix SYN-513

and that tag will be ignored.